### PR TITLE
Fix crash below android4.4 when the class cannot be found

### PR DESCRIPTION
**Pull request by deyongzhong**
If this class does not exist, devices below Android 4.4 may crash

### CODE REVIEW

File: cocos/platform/android/jni/JniHelper.cpp [hunk 1]
```diff
@@ -49,9 +49,10 @@ jclass _getClassID(const char *className) {
                                                    cocos2d::JniHelper::loadclassMethod_methodID,
                                                    _jstrClassName);
 
-    if (nullptr == _clazz) {
+    if (nullptr == _clazz || env->ExceptionCheck()) {
```
Comment (PatriceJiang) [inline cocos/platform/android/jni/JniHelper.cpp:52]: It seems that  the class is found and some exception occurs. Would you share what kind of exception did you met?

**Comment by deyongzhong** [comment]
当这个类不存在的时候，在android 4.4以下设备,_clazz不为 nullptr,但是有异常,这个是bugly上收集到的异常报告。

java.lang.ClassNotFoundException: Didn't find class "com.me2zen.remoteconfig.RemoteConfigManager" on path: DexPathList[[

**Comment by deyongzhong** [comment]
更新一下，5.1.1的设备也有这个问题。

**Comment by PatriceJiang** [comment]
https://developer.android.com/training/articles/perf-jni
>  Many JNI calls can throw an exception, but often provide a simpler way of checking for failure. For example, if NewString returns a non-NULL value, you don't need to check for an exception. However, if you call a method (using a function like CallObjectMethod), you must always check for an exception, because the return value is not going to be valid if an exception was thrown.

**Comment by holycanvas** [comment]
感谢

**Pull request merged**